### PR TITLE
haiku adding a subset of syscalls to be used by fuzzers and calls

### PR DESCRIPTION
**Pull request by devnexen**
interceptors.

### CODE REVIEW

File: libc-test/build.rs [hunk 1]
```diff
@@ -4578,6 +4578,10 @@ fn test_haiku(target: &str) {
     });
 
     cfg.skip_fn(move |name| {
+        // private system calls
```
Comment (JohnTitor) [inline libc-test/build.rs:4581]: I remember a similar discussion happened somewhere but private items shouldn't be exposed here, I think.

Comment (devnexen) [reply libc-test/build.rs:4581]: do you mean in the crate or in build.rs ?

Comment (JohnTitor) [reply libc-test/build.rs:4581]: From the crate, I mean, as it's basically unstable.

Comment (devnexen) [reply libc-test/build.rs:4581]: oh ok

**Pull request closed without merging**